src/js/letters/actions/letters.js

```diff
@@ -31,3 +31,11 @@ export function getBenefitSummaryOptions() {
     );
   };
 }
+
+export function updateBenefitSummaryOptionsStatus(propertyPath, value) {
+  return {
+    type: 'UPDATE_BENEFIT_SUMMARY_OPTIONS_STATUS',
```

Comment:
I wonder if this could just be `UPDATE_BENEFIT_SUMMARY_OPTION` and `updateBenefitSummaryOption()`